com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/shader/ShaderCompilePipeline.java

```diff
@@ -92,7 +93,7 @@ private static Integer ShaderLanguageToVersion(ShaderDesc.Language shaderLanguag
             case LANGUAGE_GLES_SM300 -> 300;
             case LANGUAGE_GLSL_SM330 -> 330;
             case LANGUAGE_GLSL_SM430 -> 430;
-            case LANGUAGE_HLSL       -> 50;
+            case LANGUAGE_HLSL       -> 51;
```

Comment:
Maybe we need HLSL_51 and so forth here. XBox requires 6.0+ i think. It would be _even_ better if we could just pass the version into the shader blob instead of having these enums with specific versions?